xnmt/modelparts/attenders.py

```diff
@@ -192,4 +196,58 @@ def calc_attention(self, state):
 
   def calc_context(self, state):
     attention = self.calc_attention(state)
-    return self.I * attention
\ No newline at end of file
+    return self.I * attention
+
+
+class FixedSizeAttSeqTransducer(transducers_base.SeqTransducer, Serializable):
```

Comment:
I had moved this to  xnmt/transducers/fixed_size_att.py because it should probably live there, maybe that got lost somehow?